app/Controller/CoGroupsController.php

```diff
@@ -440,9 +440,10 @@ function isAuthorized() {
     $p['index'] = ($roles['cmadmin'] || $roles['coadmin'] || $roles['comember']);
     $p['search'] = $p['index'];
     
-    // Nest a Group within another Group?
-    // This aligns with CoGroupNestingsController::isAuthorized
+    // Nest a Group within another Group? Delete a nested group?
+    // These align with CoGroupNestingsController::isAuthorized
     $p['buildnest'] = !$readonly && ($roles['cmadmin'] || $roles['coadmin'] || $roles['couadmin']);
+    $p['deletenest'] = !$readonly && ($roles['cmadmin'] || $roles['coadmin'] || $roles['couadmin']);
```

Comment:
Alternately, these permissions could be renamed $p['nest'] (since access to the add, edit, or delete permissions are identical from the CoGroupsController) and the current $p['nest'] could be renamed $['viewnest'] given that's how it's used. But 'buildnest' and 'deletenest' correspond directly to 'add' and 'delete' in the CoGroupNestingsController. So that may be fine enough.